string class_id
uint8 instance_id
geometry_msgs/Pose pose
geometry_msgs/Vector3 size
geometry_msgs/Vector3 min
geometry_msgs/Vector3 max